.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -200,7 +200,7 @@ validate-content-conf:
       if [[ ${MARKETPLACE_VERSION} = "xsoar" ]]; then
         echo "Creating Instances, only for XSOAR."
         [ -n "${TIME_TO_LIVE}" ] && TTL=${TIME_TO_LIVE} || TTL=300
-        time python3 ./gcp/create_instance.py --env-type "$IFRA_ENV_TYPE" --outfile "${ARTIFACTS_FOLDER_SERVER_TYPE}/env_results.json" --creds $CONTENT_BUILD_GCP --zone $GCP_ZONE
+        python3 ./gcp/create_instance.py --env-type "${IFRA_ENV_TYPE}" --outfile "${ARTIFACTS_FOLDER_SERVER_TYPE}/env_results.json" --filter-envs "${ARTIFACTS_FOLDER_SERVER_TYPE}/filter_envs.json" --creds "${CONTENT_BUILD_GCP}" --zone "${GCP_ZONE}"
```

Comment:
Why remove the `time`?

Reply:
i'm doing it in python in the create_instances.py, the time is more accurate there than bash `time`